soauth/database/group.py

```diff
@@ -41,6 +41,38 @@ class Group(SQLModel, table=True):
         link_model=GroupMembership,
         sa_relationship_kwargs=dict(lazy="joined"),
     )
+    grants: str = Field(default="")
+
+
+    def has_grant(self, grant: str) -> bool:
```

Comment:
Please apply the changes from `main` that make sure the grants are lowercase (see changes in user)